### ISSUE TITLE
VM issue about shared folders (bindfs.bind_folder)

### ISSUE THREAD

**Issue by ranjitk90** [gael-ian/vagrant-bindfs#94]
I updated my environement so I have the following configuration.

- vagrant box : generic/ubuntu1804 (virtualbox, 3.1.2)
- vagrant version : 2.2.13
- vagrant-vbguest : 0.26.0
- vagrant-bindfs : 1.1.8
- vagrant-hostsupdater : 1.1.1.160
- vagrant-share : 1.1.11
- vagrant-winnfsd : 1.4.0

While doing a **vagrant up**, Im facing a **Guest Additions** issue:

_The guest additions on this VM do not match the installed version of
VirtualBox! In most cases this is fine, but in rare cases it can
cause things such as shared folders to not work properly. If you see
shared folder errors, please update the guest additions within the
virtual machine and reload your VM._

As a consequence, it doesn't bind my folder from my Windows 10 to my Ubuntu. In the code below, only the first line is working:

```
config.vm.synced_folder ".", "/var/www/test__nfs", type: "nfs"
config.bindfs.bind_folder "/var/www/test__nfs", "/var/www/test", perms: "u=rwX:g=rwX:o=rX"
```

If I do `ls /var/www/` I can see **test__nfs** but not **test**.

I could align the version of the Guest Additions and the VM with:

`vagrant vbguest --auto-reboot`

I also updated the kernel and restarted Ubuntu:

`sudo apt upgrade sudo shutdown now`

Im still getting this issue..

I also reinstalled and reinstalled everything.

**Comment by lukequinnell** [comment]
Same issue for me to using 2.2.13.

**Comment by JoelLau** [comment]
+1 for me, except I am running macOS Catalina version 10.15.7

```
$ vagrant --version
Vagrant 2.2.13

$ vagrant plugin list
vagrant-bindfs (1.1.8, global)
```

**Comment by gael-ian** [comment]
Hello,

Can any of you add `config.bindfs.debug = true` to his Vagrantfile and post here the output of a `vagrant up`?

Thanks

**Comment by gael-ian** [comment]
This seems to be related to a bug in the latest Vagrant release (See [hashicorp/vagrant#12035](https://github.com/hashicorp/vagrant/issues/12035).

You can rollback to Vagrant 2.2.10 and wait for a fix.

I'll close this issue as it's not a vagrant-bindfs problem but feel free to reopen it if needed.

**Comment by gael-ian** [comment]
A fix has been found by Vagrant's developpers and should be released soon as 2.2.14.
